ng-dev/release/publish/test/release-notes/generation.spec.ts

```diff
@@ -28,12 +31,10 @@ describe('release notes generation', () => {
 
     releaseConfig = {npmPackages: [], buildPackages: async () => []};
     githubConfig = {owner: 'angular', name: 'dev-infra-test', mainBranchName: 'main'};
+    setConfig({github: githubConfig, release: releaseConfig});
```

Comment:
`setConfig` persists and leaks between tests. I think a _spy_ is the best solution here as it gets cleared automatically on test completion.